compiler-builtins/src/probestack.rs

```diff
@@ -112,14 +112,13 @@ pub unsafe extern "custom" fn __rust_probestack() {
             // for this target, [manually patch for LVI].
             //
             // [manually patch for LVI]: https://software.intel.com/security-software-guidance/insights/deep-dive-load-value-injection#specialinstructions
-            pop %r11
+            pop r11
```

Comment:
Something I've noticed is that LLVM automatically inserts `lfence` instructions on sgx targets. Right now, `lfence` appears twice.